packages/core/src/payment/strategies/worldpayaccess/worldpayaccess-payment-strategy.ts

```
@@ -0,0 +1,179 @@
+import { merge, some } from 'lodash';
+import { InternalCheckoutSelectors } from '../../../checkout';
+import { NotInitializedError, NotInitializedErrorType, RequestError } from '../../../common/error/errors';
+import { OrderPaymentRequestBody, OrderRequestBody } from '../../../order';
+import { PaymentArgumentInvalidError } from '../../errors';
+import { PaymentInitializeOptions } from '../../payment-request-options';
+import { CreditCardPaymentStrategy } from '../credit-card';
+import { WorldpayAccess3DSOptions, WorldpayAccessAdditionalAction, WorldpayAccessPaymentInitializeOptions } from './worldpayaccess-payment-options';
+
+const IFRAME_NAME = 'worldpay_hosted_payment_page';
+const IFRAME_HIDDEN_NAME = 'worldpay_hosted_hidden_payment_page';
+const INVALID_URL = 'invalid url';
+
+export default class WorldpayaccessPaymetStrategy extends CreditCardPaymentStrategy {
+
+    private _initializeOptions?: WorldpayAccessPaymentInitializeOptions;
+
+    async initialize(options?: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+        this._initializeOptions = options && options.worldpay;
```

Do we need to throw an error if `worldpay` is not available in options?